Structure_Engine/Query/Extrude.cs

```diff
@@ -38,6 +42,10 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Computes an extrusion of the section along the Bar centreline.")]
+        [Input("bar", "The Bar to get an extruded shape from. This requires the property of the Bar to be of a type with a section profile.")]
+        [Input("simple", "If false, a full extrusion of the section curves along the Bar will be returned. If true, a box mesh enclosing the extruded section will be returned.")]
```

Comment:
What is the box mesh in this case? Is it a bounding box? Can this be confused with Mesh, FEMesh etc.

Reply:
will be as a bounding box in local coordinates